### PR TITLE
Section for each item

### PR DESCRIPTION
**Pull request by chenbd**
This PR add a new macro `section_for_each_item` which makes code cleaner.

### CODE REVIEW

File: hypervisor/platform/platform.c [hunk 1]
```diff
@@ -19,33 +19,22 @@
...
 	struct platform *p;
-
-	pstart =(unsigned long)&__platform_start;
-	pend = (unsigned long)&__platform_end;
-	count = (pend - pstart) / sizeof(struct platform *);
-	pp = (struct platform **)pstart;
-
-	for (i = 0; i < count; i++) {
+	section_for_each_item(__platform_start, __platform_end, pp)
+	{
```
Comment (minos-project) [inline hypervisor/platform/platform.c]: the {} code style should like this:

if () {

}

for () {
}

while () {
}

Comment (chenbd) [reply hypervisor/platform/platform.c]: Yes, I know this. guess It's better for having a coding style tool like clang-format.

Comment (chenbd) [reply hypervisor/platform/platform.c]: Does this looks good?[.clang-format](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/.clang-format?h=v5.0-rc8)

Comment (minos-project) [reply hypervisor/platform/platform.c]: How to use the .clang-format file ?

Comment (chenbd) [reply hypervisor/platform/platform.c]: install clang-format on your computer.
add .clang-format to the root of the code base。

Comment (minos-project) [reply hypervisor/platform/platform.c]: Ok, could you help create a complete .clang-format file for Minos. Minos's code style should as same as Linux kernel. Thanks.

Comment (chenbd) [reply hypervisor/platform/platform.c]: this is 253616d  the style format by clang-format, please check.

**Comment by minos-project** [comment]
Kindly reminder that next time when you submit patches, please follow below rule:
 -- one topic only submit one commit.

Thanks

**Pull request merged**